rfc/2022/2022-06-15 - The permissions lifecycle for platform tools wiill be managed via version control.md

```diff
@@ -0,0 +1,109 @@
+# RFC: The permissions lifecycle for platform tools will be managed via version control
+
+* Comment Deadline: `2022-07-15`
+* Team Crew and Name: [Platform/Access Management Team](https://dsva.slack.com/archives/C02GQ27HXQW)
+* Authors:
+  * [Kevin Duensing](https://github.com/kjduensing)
+* [Original RFC Pull Request](https://github.com/department-of-veterans-affairs/va.gov-platform-architecture/pull/26)
+
+## Background
+In order to fully leverage the powerful authorization services of Keycloak, the platform tool integrated into Keycloak must be configured with a set of permissions. Permissions are composed of roles, resources, scopes, policies, and various other attributes. AMT has recommended a format and schema for defining these permissions<sub>[[RFC6]](https://vfs.atlassian.net/wiki/spaces/AMT/pages/2142011842/RFC+6+AMT+integrators+can+describe+fine-grained+permissions+using+JSON)</sub>.
+
+Currently, no holistic process has been defined to direct the flow from permissions generation to permissions integration. The intent of this RFC is to gather feedback and criticism of a proposed process for managing the lifecycle of permissions of a platform tool.
+
+## Motivation
+Though AMT has defined a schema for describing permissions, there is no prescribed process for the managing the lifecycle of a platform tool's permissions. Because of the lack of a permissions lifecycle management process, integrations with Keycloak have led to lengthy manual integrations. Defining and implementing a process using existing and popular tooling will reduce confusion, streamline the process, and pave the way for automation opportunities in the near future. 
+
+## Design
+```mermaid
+%%{init: {'theme': 'neutral'}}%%
+flowchart TD
+  init(0: AuthZ desired) --> impl(1: Describe permissions)
+  impl --> apply(2: Apply permissions)
+  apply --> use(3: Use permissions)
+  use --> revise(4: Revise permissions)
+  revise --> impl
+```
+
+### 0: Authorization Desired 
+This step handles the cases:
+1. The application has not been integrated into Keycloak
```

Comment:
how are applications identified within keycloak?

Reply:
info is potentially available w/in console. what's the integration lift?